config/vpc/config.go

```diff
@@ -13,5 +16,22 @@ func Configure(p *config.Provider) {
 				"name",
 			},
 		}
+		// Delete deprecated fields
+		delete(r.TerraformResource.Schema, "name")
```

Comment:
Does the line 14-18 still need to retain because of there has deleted name from its schema?